plugins/wacom-raw/wacom-raw.quirk

```diff
@@ -134,4 +134,4 @@ GType = FuWacomEmrDevice
 GType = FuWacomAesDevice
 WacomI2cFlashBlockSize = 128
 WacomI2cFlashBaseAddr = 0x8000
-WacomI2cFlashSize = 0x24000
+WacomI2cFlashSize = 0x28000
```

Comment:
is this *all* AES panels?

Reply:
available address for the latest AES, G14T, is expanded to 0x4000, so the maximum size is also required a change.
If this plug-in writes the firmware up to this address, the other generations than G14T, sends back an  error in order not to overwrite the address on them.

Is this current wacom-raw plugin to write up to "WacomI2CFlashSize" ? If so, I guess I have to modify the write part as well.
Then, I'd like to know where I should make modification on because chunks->len seems to be set somewhere else.